openshift/templates/backups/README.md

```diff
@@ -56,4 +56,43 @@ gzip -cd ~/backups/db/daily/test.sql.gz | mysql -u $WORDPRESS_DB_USER -p$(cat $M
 * Use the sidecar, see instructions for deploying the sidecar.
 * Do a ```rsync -a --exclude=restore /home/sidecar/backups/html/ /var/www/html/``` 
     * you might have to delete the files in /var/www/html, but **WARNING** make sure you know what you are doing.
-* Switch from backup volume to regular volume as indicated below, so now WordPress should be using restored files.
\ No newline at end of file
+* Switch from backup volume to regular volume as indicated below, so now WordPress should be using restored files.
+
+### Manual backups/site duplication
+At times you may wish to back up the database manually - for example, when copying a site or in preparation for a potentially disruptive operation.
+1. Export the database by running the following command from the `sidecar of the original site`. Replace example.gov.bc.ca with the URL of the site.
+    ```bash
+    wp db export --tables=$(wp db tables --url=old-example.url/ --path=/var/www/html/ --all-tables-with-prefix --format=csv) --url=old-example.url/ --path=/var/www/html/ backup.sql
+    ```
```

Comment:
```suggestion
    ```bash
    wp db export \
    --tables=$(wp db tables --url=old-example.url/ --path=/var/www/html/ --all-tables-with-prefix --format=csv)\
     --url=old-example.url/\
     --path=/var/www/html/ backup.sql
```

I don't really understand what the part of the command inside the `$()` really does, 
but it seems like global parameters are being used to assemble a list of tables in CSV format as an argument to the `wp db export --tables` command...

Reply:
Close. We're using $() to run a subshell which executes the wp db tables command, which outputs a list of the tables in the database matching the parameters we feed it(--url, --path, --all-tables-withprefix and --format).

This lets us use command substitution to dynamically generate the --tables parameter in the original wp db export command. Hopefully that makes sense!

https://unix.stackexchange.com/questions/147420/what-is-in-a-command
https://developer.wordpress.org/cli/commands/db/tables/